### DISCUSSION TITLE
HTTPS for Hydrogen Music Website

### DISCUSSION THREAD

**Discussion by dabfalcon** [hydrogen-music/hydrogen#1718] [General]
Hi all, hope you are going well!

I was just on 'https://hydrogen-music.org/' and found that it doesn't load properly due to no SSL certificate and thought I should raise it in case the maintainer is unaware. I hope this is the right place to raise it and hope they can get some free SSL like LetsEncrypt.

Cheers

**Comment by theGreatWhiteShark** [comment]
Hey @dabfalcon ,

There is already an issue covering this problem https://github.com/hydrogen-music/hydrogen-music/issues/9 . Let's hope we get all the pieces together soon to fix it

**Reply by dabfalcon** [reply]
Ah thanks, a duplicate then. Good luck!
